bazel/external/requirements.txt

```diff
@@ -979,7 +975,7 @@ oslo-config==8.8.0 \
     --hash=sha256:96933d3011dae15608a11616bfb00d947e22da3cb09b6ff37ddd7576abd4764c \
     --hash=sha256:b1e2a398450ea35a8e5630d8b23057b8939838c4433cd25a20cc3a36d5df9e3b
     # via -r requirements.in
-oslo-i18n==5.1.0 \
+oslo.i18n==5.1.0 \
```

Comment:
It seems some naming scheme on pypi has changed?

Reply:
yes, I noticed too - lets see if ci is ok with this